### ISSUE TITLE
Add analyzer for `Explicit` without `Test` attribute

### ISSUE THREAD

**Issue by cbersch** [nunit/nunit.analyzers#916]
Would be nice, if the following code would raise a diagnostic that `Explicit` attribute makes no sense without also specifying `Test`:

```csharp
public sealed class MyTest
{
    [Explicit]
    public void Invalid()
        => Assert.That(12, Is.EqualTo(12));

    [Test]
    [Explicit]
    public void Valid()
        => Assert.That(12, Is.EqualTo(12));
}
```

**Comment by cbersch** [comment]
I guess that should be enough, thanks
